api-report/map.api.md

```diff
@@ -94,7 +94,9 @@ export interface IDirectoryEvents extends IEvent {
     (event: "containedValueChanged", listener: (changed: IValueChanged, local: boolean, target: IEventThisPlaceHolder) => void): any;
     (event: "subDirectoryCreated", listener: (path: string, local: boolean, target: IEventThisPlaceHolder) => void): any;
     (event: "subDirectoryDeleted", listener: (path: string, local: boolean, target: IEventThisPlaceHolder) => void): any;
+    (event: "subDirectoryUndeleted", listener: (path: string, local: boolean, target: IEventThisPlaceHolder) => void): any;
```

Comment:
does this need to be different than create? in shared string i believe the rollback of an insert is a delete, and the rollback of a delete is an insert.

Reply:
The initial idea was that the assumptions made about the state of the directory when a create event is emitted are not necessarily the same as the assumptions one would make when a delete is rolled back, since the directory could have data stored in it (instead of being a completely new, empty directory). If it's more beneficial to simplify the events and just add documentation to clarify what happens on the rollback of a delete though, we can do that instead.

Reply:
We already use event subDirectoryCreated in shared directory on rollback of delete. Is there any problem with that? I think we should keep the events to a minimum if possible. Is there an issue with keeping the same event? If somebody is using rollback feature, then they should assume that if they are rolling back a delete than the created directory could have data inside it? Just an opinion.